infra/src/main/java/com/redhat/developer/infra/utils/exceptions/CloudEventSerializationException.java

```diff
@@ -0,0 +1,12 @@
+package com.redhat.developer.infra.utils.exceptions;
+
+public class CloudEventSerializationException extends RuntimeException {
```

Comment:
Should this extend `IngressException`?

Reply:
The idea of the CloudEventUtils and its related exceptions (CloudEventSerializationException, CloudEventDeserializationException) is that they are re-usable in different modules (`ingress`, `executor` and others if needed). According to this design, this class cloud not extend `IngressException`, wdyt?

Reply:
It depends.. some people do not like checked exceptions and some others do. 

My personal preference is always unchecked exception. But if you think this is a good fit for a checked exception, I can change it. Let me know

Reply:
Let's not worry about it for now 👍